src/coreclr/vm/qcallentrypoints.cpp

```diff
@@ -310,6 +310,8 @@ static const Entry s_QCall[] =
     DllImportEntry(LogContentionLockCreated)
     DllImportEntry(LogContentionStart)
     DllImportEntry(LogContentionStop)
+    DllImportEntry(LogWaitHandleWaitStart)
+    DllImportEntry(LogWaitHandleWaitStop)
```

Comment:
These are unnecessary since the event is not raised from the managed side in CoreCLR currently